docs/api/v3.rst

```diff
@@ -227,6 +227,35 @@ Project create
     :statuscode 400: Some field is invalid
 
 
+Project update
+++++++++++++++
+
+.. http:patch:: /api/v3/projects/(string:project_slug)/
+
+    Edit an existent project.
```

Comment:
```suggestion
    Edit an existing project.
```

Reply:
We should also use `edit` or `update` here. The title says `update` but this says `edit`.